nest-forms-backend/src/utils/files.ts

```diff
@@ -1,5 +1,50 @@
+import { GenericObjectType } from '@rjsf/utils'
+import * as jwt from 'jsonwebtoken'
+import { mapValues } from 'lodash'
+
+import { FormWithFiles } from './types/prisma'
+
 // PDF_FORM_FAKE_FILE_ID is used to tell GinisService functions that they shouldn't look for this file in database
 export const PDF_FORM_FAKE_FILE_ID = 'pdf-form-fake-id'
 export const fileIdIsValid = (fileId: string): boolean =>
   fileId !== PDF_FORM_FAKE_FILE_ID
 export const PDF_EXPORT_FILE_NAME = 'pdf-export.pdf'
+
+export const getFileIdsToUrlMap = (
```

Comment:
Is there a reason no to extract it to FilesService? It would get all the variables (secret, self url) injected (which is the default way of getting them).

Reply:
I guess it would be fine in FilesHelper, looking over it there are similar functions already. I'll move them so that I don't create more disparate approaches in codebase.

Not a fan though. On principle, I'd prefer having functions like this outside of services, injecting a couple config variables (as opposed to, for example, working with database or network calls) shouldn't suddenly stop a simple transformation from being a pure function.

Reply:
Dug slightly deeper and talked in person afterwards. Since I don't want to delve into Files module refactor keeping this as is for now.